roles/postgres/tasks/main.yml

```diff
@@ -112,6 +105,70 @@
     postgres_sslmode: "{{ pg_config['resources'][0]['data']['sslmode'] | default('prefer'|b64encode) | b64decode }}"
   no_log: true
 
+- name: Set database as managed
+  set_fact:
+    managed_database: "{{ pg_config['resources'][0]['data']['type'] | default('') | b64decode == 'managed' }}"
+
+- name: Set postgres label if not defined by user
+  set_fact:
+    postgres_label_selector: "app.kubernetes.io/instance=postgres-{{ ansible_operator_meta.name }}"
+  when: postgres_label_selector is not defined
+
+- name: Get the old postgres pod information
+  k8s_info:
+    kind: Pod
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    label_selectors:
+      - "{{ postgres_label_selector }}"
+      - app.kubernetes.io/version: '12'
```

Comment:
I'm a little confused with the logic as the version & value seem to be a new label.

Reply:
@chambridge I am trying to filter for only the old postgresql pod here by this label: https://github.com/pulp/pulp-operator/blob/main/roles/postgres/templates/postgres.yaml.j2#L18 (which should be 12 for the old postgres pod).

Reply:
Gotcha. I was confused by the new line here: https://github.com/pulp/pulp-operator/pull/400/files#diff-e953e7a3d1f04931f56d660f00c5bf9aab9e7f88ab3ece25dc6f9367537ad6f6R45